src/errors.rs

```diff
@@ -59,21 +59,10 @@ pub enum Error {
         #[cfg(feature = "backtraces")]
         backtrace: snafu::Backtrace,
     },
-    #[snafu(display("Invalid {}: {}", field, msg))]
```

Comment:
Please revert the changes to this file.

Invalid is intended for use inside contracts, and I think I demo that in some (or meaning to add that as soon as I get back to contract coding)

Reply:
The motivation here was to avoid the naming overlap between cosmwasm and cosmwasm_vm, which is somewhat hard to follow. Maybe we can differentiate the names a little bit, since they seem to be entirely different things. Ideas? What's the intend of `cosmwasm::Errors::ValidationErr`?

Reply:
I was going through the contracts, figuring out where there was too much boilerplate that I could reduce, and making some helpers for this (like `contract_err`, `cw-storage`, etc). I noticed your [very correct validation in the erc20 contract](https://github.com/confio/cosmwasm-examples/blob/master/erc20/src/contract/mod.rs#L102-L119) and I wanted to make a helper. It could just be `contract_err`, but I figured this was part of a general class, and we could add some more structure.

I am happy for another name (like `InvalidData`). Or do you not see the need for a specific error type here?

Reply:
I'll remove the commit that deletes `cosmwasm::Errors::ValidationErr` from this PR. There is room for improvement but there are more important things open, so ignoring as out of scope.